src/arch/x86/x86_64/src/lib.rs

```diff
@@ -3,13 +3,44 @@
 #![feature(global_asm)]
 
 const PAGE_SIZE: usize = 4096;
-use rpp_procedural::preprocess_asm;
 
 pub mod acpi;
 pub mod bzimage;
 pub mod consts;
 pub mod ioport;
 
+#[macro_export]
+macro_rules! const_asm {
+    ($asm:expr, $(options($($option:ident),*))? $(,)?) => {
+        global_asm!(
+            $asm,
+            pse = const $crate::consts::x86::X86_CR4_PSE,
```

Comment:
would calling it CR4_PSE help? Danny?

Reply:
I would be in favor of line 17 basename being exactly the same on both sides.

But in the definition of the constant itself (elsewhere), the `X86` in `x86::X86_CR4_PSE` is redundant anyway--so, sure, why not rename the latter too.

But really, I want to stress that this PR is pretty good as-is and if @retrage wants to do this stuff later or not at all, that's fine, too.

I've just been at the receiving end of one too many pointless rename-the-imports--where everyone then has to spend time mapping what is what using grep manually before one understands the source code.